llvm-project/stage3.cmake

```diff
@@ -0,0 +1,85 @@
+# Please try to keep these cmake variables alphabetically sorted.
+
+# Enable optimizations, as opposed to a debug build.
+set(CMAKE_BUILD_TYPE "Release" CACHE STRING "")
+
+# Use stage1's clang to bootstrap stage2, as opposed to Alpine's clang (which
+# shouldn't be installed on the system at this point, but would be installed to
+# /usr/bin/clang).
+#set(CMAKE_CXX_COMPILER "/usr/local/bin/clang++" CACHE FILEPATH "")
+
+# Use libc++ from stage1.
+# We could not set CLANG_DEFAULT_CXX_STDLIB in stage1 because Alpine doesn't
+# distribute libc++ ATM.
+# Even though we later will set LLVM_ENABLE_LIBCXX, cmake will fail thinking
+# that "The C++ compiler is not able to compile a simple test program." As with
+# stage1, this is an unfortunate effect of Alpine packaging the stdlibc++
+# headers in the g++ package, rather than the libstdc++ package. In stage1, we
+# used those headers; in stage2 we do not.
+#set(CMAKE_CXX_FLAGS "--stdlib=libc++" CACHE STRING "")
+
+# See above comment for CMAKE_CXX_COMPILER.
+#set(CMAKE_C_COMPILER "/usr/local/bin/clang" CACHE FILEPATH "")
+set(CMAKE_C_FLAGS "--sysroot=/sysroot" CACHE STRING "")
+set(CMAKE_CXX_FLAGS "--sysroot=/sysroot" CACHE STRING "")
+
+# Use libunwind from stage1.
+# Statically link resulting executable.
+set(CMAKE_EXE_LINKER_FLAGS "-static -lc++abi" CACHE STRING "")
+#set(CMAKE_SHARED_LINKER_FLAGS "--unwindlib=libunwind" CACHE STRING "")
```

Comment:
oops, I mean to delete some of these. I need to update the comments in this file, too. It was copied from stage2.cmake.